docs/getting-started/products/yvaults/v3.md

```diff
@@ -0,0 +1,245 @@
+# Vaults Version 3
+
+Version 3 yVaults iterates on Version 2 by increasing robustness and developing yearn’s path towards further decentralization, while keeping the same proven product (yield-bearing tokens) that abstract builders and users from the hard work of yield farming. Version 3 will initially have the same functionality as Version 2, but the vaults are more robust and modular.
+
+Version 3 yVaults improves on Version 2 by:
+- Increasing vault modularity, allowing for smaller and safer pieces of code.
+- Simplify strategy creation, empowering strategists and reducing the chance for errors.
+- Adding new products like ySwaps which increases swap efficiency and yJuniorTranches which allows for different risk profiles.
+
+## Smart Modules
+
+Smart Modules implement logic to be iterated until it can be made immutable. If any Smart Module fails, the vault can live without them just enough to return funds to depositors. **When Version 3 launches, the Smart Modules will replicate Version 2 vault behavior.**
```

Comment:
`to be iterated until it can be made immutable`
Not really sure here sounds odd though.

Reply:
```suggestion
Smart Modules implement core vault logic that will be iterated until they can be made immutable. If any Smart Module fails, the vault can live without them just enough to return funds to depositors. **When Version 3 launches, the Smart Modules will replicate Version 2 vault behavior.**
```

Reply:
@DarkGhost7 sounds better?